test/lexer/comments.txt

```diff
@@ -5,8 +5,8 @@
 
 // Another comments
 
-int main(){
+int main() {
     // Just another comment 
     // Basically a hello world program, Peace!
     cout << "Hello";
-}
\ No newline at end of file
+}
```

Comment:
I think the earlier version was better as it portrayed that an un-sanitized input file.

Reply:
ok will fix this